spec/13_input_output_spec.rb

```diff
@@ -4,21 +4,23 @@
 
 # rubocop:disable Layout/LineLength, Metrics/BlockLength
 
-# Ruby code that was written before you learned how to use rpsec, may be nearly
+# Ruby code that was written before you learned how to use RSpec may be nearly
 # impossible to test. For example, in the 13_input_output file, there are two
-# identical games - ImpossibleToTestGame and NumberGame. Take a look at both
-# game and look for the differences that may make it easier or harder to test.
+# identical games: ImpossibleToTestGame and NumberGame. Take a look at each
+# game and look for differences that may make one easier or harder to test
+# than the other.
 
-# One key difference is that NumberGame has smaller, isolated methods.
+# One key difference between the two is that NumberGame has smaller,
+# isolated methods.
 
 # Small and isolated methods that only do one thing are easier to test.
 # Long methods are like a run-on sentence that should have been divided into 2 or 3 different sentences so that everything could be clearly understood and in this case if a method does many different things it can be difficult to test.
 
-# Therefore, if you are new to testing, be open to refactoring your previous
-# code to make writing testing easier. As you learn testing, you are also
-# learning how to write better testable methods.
+# So if you are new to testing, be open to refactoring your previous
+# code, to make writing tests easier. As you learn testing, you wil also
```

Comment:
Everything looks amazing so ar! Please change 'wil' to be 'will'.

Reply:
Thanks for catching that! Fixed.